### PR TITLE
feat: Add new project overview document

### LINKED ISSUE #556: Add a 'Project overview' page
As a contributor I want to get an overview of the repositories that are used to create the SDP, which one does what, which ones I need and what the intentions are behind them.

The new page should contain information from the [Development Dashboard](https://docs.stackable.tech/home/stable/contributor/development-dashboard) and replace it. The first table can be removed, and the second should be turned into more free text. Additionally there should be info about the `stackable-cockpit`.

make a diagram how templating flows into the repos, how operator-rs is pulled in, where images are build for the products, where images are stored and how it is pulled etc. A high level overview

### PR DESCRIPTION
**Pull request by fhennig**
fixes #556 

preview: https://deploy-preview-576--stackable-docs.netlify.app/home/nightly/contributor/project-overview

the new project overview should contain most info from the developer dashboard page that it replaces. Some of the other links are now just found in the READMEs of the operators.

### CODE REVIEW

**Comment by fhennig** [comment]
Oh, I forgot stackable cockpit and stackablectl!

File: modules/contributor/pages/project-overview.adoc [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+[[product-artifacts]]
+=== Product artifacts
+
+A lot of artifacts are stored in the https://repo.stackable.tech/#browse/browse[Nexus repo].
+Under `packages` you find product binaries, the different `helm-*` stores contain the Helm Charts and Docker images are stored there too.
+
+[[docker-images]]
+=== Docker images
+
+Docker images are stored in Nexus as mentioned above, but also in https://oci.stackable.tech - which is where we want to move to in the future.
```
Comment (razvan) [inline modules/contributor/pages/project-overview.adoc:94]: Release images are also distributed to quay.io and in the future maybe also hub.docker.com

**Review by razvan — APPROVED** [review]
A very nice idea and a good start. Just added a minor comment.

File: modules/contributor/pages/project-overview.adoc (hunk 1 above)
Comment (fhennig) [reply modules/contributor/pages/project-overview.adoc:94]: Or is there a way to actually pull from quay? Happy to have a follow-up PR to update this section.

**Pull request merged**